src/components/network-map-viewer/network/network-map.jsx

```diff
@@ -523,6 +528,118 @@ const NetworkMap = (props) => {
         setCentered(INITIAL_CENTERED);
     }, [mapLib?.key]);
 
+    const [features, setFeatures] = useState({});
```

Comment:
What is the meaning of `features` in `NetworkMap` context ? Maybe choose a name with more precision ?

Reply:
I saw it is named like that in react map gl or mapBox or mapLibre OK.
Maybe add some precision here somehow. `selectedLayerFeatures` ?